src/util/util_env.cpp

```diff
@@ -157,6 +157,10 @@ namespace dxvk::env {
             || getExecutableName() == std::string("tlou-i-l.exe");
     }
 
+    bool isTheLastOfUsPartTwo() {
+        return getExecutableName() == std::string("tlou-ii.exe");
```

Comment:
Any idea if there is also an alternative name, like part one had?

Reply:
Right, found another one on SteamDB, should be more foolproof now.